src/table.ts

```diff
@@ -1203,17 +1200,31 @@ class Table extends common.ServiceObject {
    * });
    */
   createLoadJob(
-    source: string | File,
+    source: string | File | File[],
     metadataOrCallback?: JobLoadMetadata | JobCallback,
     cb?: JobCallback
-  ): void | Promise<JobResponse> | Writable {
+  ): void | Promise<JobResponse> {
     const metadata =
       typeof metadataOrCallback === 'object' ? metadataOrCallback : {};
     const callback =
-      typeof metadataOrCallback === 'function'
-        ? metadataOrCallback
-        : cb || common.util.noop;
+      typeof metadataOrCallback === 'function' ? metadataOrCallback : cb;
+
+    this._createLoadJob(source, metadata).then(
```

Comment:
Can we bring the functionality from `_createLoadJob` into this function?

Reply:
that's how it was before, i did this to minimize callback usage in the main body of the logic.